theseus/optimizer/tests/test_manifold_gaussian.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import copy
+
+import numpy as np
+import pytest  # noqa: F401
+import torch
+
+import theseus as th
+from theseus.optimizer.manifold_gaussian import ManifoldGaussian
```

Comment:
You should add `ManifoldGaussian` to the appropriate inits to access from `th.` directly.